webapp-django/crashstats/signature/jinja2/signature/signature_reports.html

```diff
@@ -26,6 +26,8 @@
                     {% if crash[column] %}
                         {% if column == 'date' %}
                             {{ crash[column] | human_readable_iso_date }}
+                        {% elif column == 'install_time' %}
+                            {{ crash[column] | timestamp_to_date }}
```

Comment:
Is this **ALWAYS** a value that can be turned into a datetime? I mean, is this value something we can trust to always be sane? If it comes from breakpad is could be weird can cause problems with `datetime.datetime.fromtimestamp`